docs/openshift_mc.md

```diff
@@ -0,0 +1,66 @@
+# Pre-Requisites for OpenShift 4.4 and higher
+
+## Preparing worker nodes
+Perform these steps before installing PSO:
+
+### 1. Perform this step to ensure iSCSI connectivity, when using RHEL OS, for each worker node.
+If using RHCOS or if the packages are already installed, continue to the next step.
+
+RHEL 7.x:
```

Comment:
Is there any particular reason only `RHEL 7.x` is listed here? Or maybe RHEL8 has these packages installed by default?

Reply:
Good point - I will have to think about this a bit more.